src/components/FoodItem.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+
+export default class FoodItem extends Component {
+    render() {
+     const {singleItem} = this.props
```

Comment:
instead of destructuring the whole item, you could destructure the specific properties, like image, calories and quantity.